src/components/configurable-columns/configurable-columns.scss

```diff
@@ -0,0 +1,18 @@
+.carbon-configurable-columns{
+  .carbon-configurable-columns__columns-wrapper {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  .carbon-form__buttons {
```

Comment:
As far as I could tell, there isn't a way to position additional action buttons passed into the `Form` component using a prop or anything like that. Happy to stand corrected.